build.py

```diff
@@ -13,5 +13,9 @@
     os.system("conan package .. --build_folder=../")
     print(os.listdir("."))
     print(os.listdir("./lib"))
+    os.system("cd .. && conan export user/channel") # This will improve next 0.25 with an --cwd=.. arg
+    os.system("conan package_files Test/0.1@user/channel -f")  # -f in case it exists, overwrite
+    os.system("conan search Test/0.1@user/channel") # To show the package binary
+    # os.system("conan upload Test/0.1@user/channel --all -r=")
```

Comment:
Beware of that imports, it might not be doing what you want.
The ``package_info()`` method of recipes define the build information for every package. It is translated to the different variables in generated files, like ``conanbuildinfo.cmake``. Please have a look in it. You will get variables like ``CONAN_LIBS_DIRS`` which will will be used to link and have the full paths to the local conan cache, not to the imported files in user space. 

Why do you want to copy them into client's workspace?